tango/test_context.py

```diff
@@ -270,12 +323,74 @@ def __enter__(self):
         """Enter method for context support."""
         if not self.thread.is_alive():
             self.start()
-        return self.device
+        return self
 
     def __exit__(self, exc_type, exception, trace):
         """Exit method for context support."""
         self.stop()
 
+# Single device test context
+
+class DeviceTestContext(MultiDeviceTestContext):
+    """ Context to run a device without a database."""
+
+    def __init__(self, device, device_cls=None, server_name=None,
+                 instance_name=None, device_name=None, properties=None,
+                 db=None, host=None, port=0, debug=3,
+                 process=False, daemon=False, timeout=None):
+        """Inititalize the context to run a given device."""
```

Comment:
As above, suggest this is removed, and class-level docstring is used instead.

Reply:
Done in 43ce4798.